packages/ui-extensions-react/src/surfaces/customer-account/hooks/buyer-identity.ts

```diff
@@ -73,7 +75,7 @@ export function usePhone<
  */
 export function usePurchasingCompany<
   Target extends RenderExtensionTarget = RenderExtensionTarget,
->(): PurchasingCompany | undefined {
+>(): OrderStatusPurchasingCompany | PurchasingCompany | undefined {
   const buyerIdentity = useApi<Target>().buyerIdentity;
```

Comment:
this one we might be able to do better un the return type: 
`Target extends OrderStatusExtensionTarget ? OrderStatusPurchasingCompany | undefined: PurchasingCompany | undefined`

Reply:
the `PurchasingCompany | undefined` from customer account standard api side